txpool/txpool.go

```diff
@@ -881,6 +898,49 @@ func (p *TxPool) resetAccounts(stateNonces map[types.Address]uint64) {
 	}
 }
 
+// updateUnadoptedCounts update the accounts' unadopted counts,
+// the number of the consecutive blocks that doesn't have the account's transactions
+func (p *TxPool) updateUnadoptedCounts(latestActiveAccounts map[types.Address]uint64) {
+	p.accounts.Range(
+		func(key, value interface{}) bool {
+			address, _ := key.(types.Address)
+			account, _ := value.(*account)
+
+			if _, ok := latestActiveAccounts[address]; ok {
+				account.unadopted = 0
+			} else {
+				account.unadopted++
+			}
```

Comment:
Nitpick: make this a method of `account`

Reply:
Added 1f9344761d8c0676269fbf20d774528236c7f6c0